kv/kvserver: TestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected failed

kv/kvserver.TestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected [failed](https://tanzanite.cluster.engflow.com/invocations/default/19c59fd8-74b2-488c-98d1-c5bf93a4d383?testReportRun=11&testReportShard=11&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on master @ [b3fec61ca90095c664f6432af864f18e9946f8bb](https://github.com/cockroachdb/cockroach/commits/b3fec61ca90095c664f6432af864f18e9946f8bb):


```
=== RUN   TestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected1542221226
    test_log_scope.go:76: use -show-logs to present logs inline
    client_merge_test.go:3060: condition failed to evaluate within 45s: from client_merge_test.go:3062: lhs not destroyed
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected1542221226
--- FAIL: TestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected (52.05s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=11</code>
 - <code>shard=11</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45156

--- a/pkg/kv/kvserver/client_merge_test.go
+++ b/pkg/kv/kvserver/client_merge_test.go
@@ -3000,7 +3000,9 @@ func TestStoreRangeMergeAbandonedFollowersAutomaticallyGarbageCollected(t *testi
 	tc := testcluster.StartTestCluster(t, 3,
 		base.TestClusterArgs{
 			ReplicationMode: base.ReplicationManual,
-			ServerArgs:      base.TestServerArgs{},
+			ServerArgs: base.TestServerArgs{
+				Settings: st,
+			},
 		})
 	defer tc.Stopper().Stop(ctx)
 	scratch := tc.ScratchRange(t)

